stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -364,6 +345,6 @@ spec:
       {{- end }}
       {{- end }}
 {{- else }}
-        - name: datadir
-          emptyDir: {}
+      - name: datadir
```

Comment:
This should probably be undone as now datadir doesn't line up with configdir above

Reply:
You are right, my IDE mislead me here.